DISCLAIMER.md

```diff
@@ -13,21 +13,23 @@ Simple Icons asks that its users read this disclaimer fully before including an
 
 > NOTE: the addition of licenses to Simple Icons is an ongoing project. Hence, the absence of licence data for a particular icon does not imply that the icon is not released under a license.
 
-Simple Icons provides data on the license under which icons are available. We ask users to carefully consider this when using an icon. Users of Simple icons are expected to request permission to use a brand's icon when necessary. As licenses are subject to change we also ask our users to regularly check if the license of the icons they use have been changed.
+> NOTE: Simple Icons is released under CC0 - though that doesn't mean to imply that all icons within the project are also CC0. Please see individual licenses where available.
+
+Simple Icons provides data on the license under which icons are available. We ask users to carefully consider this when using an icon.  As licenses are subject to change we also ask our users to regularly check if the license of the icons they use have been changed.
 
 The license under which an icon is available can be found in the icon's entry in the JSON data file in the NPM and Packagist packages.
 
 As we rely largely on our community to help us keep Simple Icons up-to-date, the license data for an icon may be outdated. If you find that the license data provided is outdated, please [submit an issue] to [the Simple Icons GitHub repository].
 
 If an icon includes a registered trademark (`®`) or trademark symbol (`™`) the recommendations outlined in [the Simple Icons Contributing Guidelines] are followed to decide whether to include the symbol or not.
 
-Simple Icons cannot be held responsible for any legal activity raised by a brand over the use of their icon through Simple Icons.
+Simple Icons cannot be held responsible for any legal activity raised by a brand, or users of the package. We ask that our users seek the correct permissions to use the icons relevant to their project.
 
 ## Brand Guidelines
 
 > NOTE: the addition of guidelines to Simple Icons is an ongoing project. Hence, the absence of guidelines for a particular brand does not imply that the brand has no guidelines.
```

Comment:
One last little change and I think this should be good to go once we get the email address set up: I think it might be a good idea to include a note here that while we work on adding guidelines for all existing icons that users should also check the source URL as it can often be the a link to the guidelines.

Reply:
Good shout. Added it onto the note line on guidelines.

Reply:
One minor nitpick regarding 18cb2be3cd4140b3e24dd33c701785205fecb1e9, I don't think the newline should be there.

One minor nitpick regarding 61c08d6868fff7055ac098982c52d23927ece007, I think we should switch the sentence around. So instead of "Users of Simple Icons should do this, in the absence of guidelines", write it as "In the absence of guidelines, users of Simple Icons should do this".

Reply:
Try that one out. Should have both your suggested edits in.